.github/workflows/compile-tests.yml

```diff
@@ -50,8 +50,15 @@ jobs:
       - name: Install wasm32-wasip1 target for Rust
         uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
+          toolchain: nightly
```

Comment:
that shouldn't be needed for wasip1, it's already tier2 in Rust which makes it available in the stable toolchain.

Reply:
And indeed, having made that change, the wasip1 build is now looking for a nightly toolchain.  Annoying!